azure-toolkit-libs/azure-toolkit-springcloud-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/utils/TextUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.toolkit.lib.common.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.fusesource.jansi.Ansi;
+import org.fusesource.jansi.Ansi.Color;
+
+import java.util.Objects;
+
+public class TextUtils {
```

Comment:
com.microsoft.azure.tools.common.util.TextUtils

Reply:
we re-designed the package structure.
and `com.microsoft.azure.tools.common.util.TextUtils` should be removed and use this one